# Data Science Tools and Ecosystem

![screenshot_1](Capturar1.PNG)

# In this notebook, Data Science Tools and Ecosystem are summarized.

![Screenshot_2](Capturar2.PNG)

# Some of the popular languages that Data Scientists use are:

1: Python
2: R
3: Julia

![Screenshot3](Capturar3.PNG)

# Some of the commonly used libraries used by Data Scientists include:

1: Pandas
2: numpy
3: scikit-learn

![Screenshot4](Capturar4.PNG)

# Data Science Tools

1: Jupyter Notebooks
2: RStudio
3: Apache Spark

![ScreenShot5](Capturar5.PNG)

# Below are a few examples of evaluating arithmetic expressions in Python

In [19]:
### x = 5
### y = 10
### z = y - x
### print(z) == 5

In [20]:
### x = 5
### y = 10
### z = x * y
### print(z)  == 50

In [23]:
### x = (3*4)+5
### print(x)

![Screenshot6](Capturar6.PNG)

# Create a code cell to convert minutes to hours

In [24]:
minutes = 200

hours = minutes / 60
print(hours)



3.3333333333333335


### The code defines minutes as 200.
### It then converts the minutes to hours by dividing by 60 (minutes / 60), because 1 hour has 60 minutes.
### The result (which will be 3.333... hours) is printed using print(hours).

![Screenshot7](Capturar7.PNG)

# Objectives

1: Code and Data Analysis
2: Markdown for Applications and Annotations
3: Data Visualization

![Screenshot8](Capturar8.PNG)

## AUTHOR: JOÃO VICTOR ARAUJO DE SOUZA

![Screenshot9](Capturar9.PNG)